Update aad-security-configure-tutorial.md (#9060)
MicrosoftDocs/sql-docs: azure-sql/managed-instance/aad-security-configure-tutorial.md

### Create an Azure AD user and create a sample table

> [!NOTE]

> > There are some limitations when the Azure AD user, that is part of the Azure AD group used as the Azure AD admin on SQL Managed Instance, logs in to SQL Managed Instance.

Reviewer: ```suggestion
> There are some limitations when the Azure AD user, which is part of the Azure AD group used as the Azure AD admin on SQL Managed Instance, signs into SQL Managed Instance.
```
